### PR TITLE
Add memberships

### PR DESCRIPTION
**Pull request by seeARMS**
Add new memberships page.

---

## PR-Codex overview
This PR adds a new feature for creating memberships on the Paragraph newsletter. 

### Detailed summary
- Added `public/img/membership.png`.
- Modified `src/pages/_app.jsx`.
- Added `src/pages/docs/memberships.md` with content about creating memberships.
- Added a membership image and description.
- Added instructions for creating paid memberships and connecting with Stripe.
- Mentioned future improvements for memberships.
- Added information about importing paid subscribers.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: src/pages/docs/memberships.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+title: Memberships
+description: Monetize your Paragraph newsletter with recurring memberships.
+---
+
+Create memberships to offer benefits to your subscribers and optionally begin monetizing with recurring payments.
```
Comment (fred-paragraph) [inline src/pages/docs/memberships.md:6]: Just an idea, not sure if it's better necessarily.

```suggestion
Create free or paid memberships to offer benefits to your subscribers and begin monetizing with recurring payments.
```

File: src/pages/docs/memberships.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+---
+
+# Memberships on Paragraph
+
+Paragraph enables you to create _memberships_ for your newsletter. Memberships allow you to grant additional benefits and perks to your subscribers.
+
+All Paragraph publications have a single free membership by default, but you can optionally create paid memberships. Upon subscribing to your newsletter, readers will be auto-join the free membership but can choose to upgrade to a paid tier (if you have any paid tiers). At any point, a subscriber can also click "Upgrade to Paid" on your publication and purchase a paid subscription.
```
Comment (fred-paragraph) [inline src/pages/docs/memberships.md:14]:
```suggestion
All Paragraph publications have a single free membership by default, but you can optionally create paid memberships. Upon subscribing to your newsletter, readers will auto-join the free membership but can choose to upgrade to a paid tier (if you have any paid tiers). At any point, a subscriber can also click "Upgrade to Paid" on your publication and purchase a paid subscription.
```

File: src/pages/docs/memberships.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+For each membership, you can upload a mmebership image and choose additional benefits and perks that are associated to each. For example, you can offer a private Discord community to a paid membership tier. You can also gate exclusive content on Paragraph to certain memberships.
```
Comment (fred-paragraph) [inline src/pages/docs/memberships.md:16]:
```suggestion
For each membership, you can upload a membership image and choose additional benefits and perks that are associated to each. For example, you can offer a private Discord community to a paid membership tier. You can also gate exclusive content on Paragraph to certain memberships.
```

File: src/pages/docs/memberships.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+After conneting with Stripe you'll be able to create paid memberships and charge a monthly or yearly fee to join.
+
+## Future improvements
+
+This is the first iteration of what's to come. Within the next few months, memberships will be:
+
+* Backed by NFTs. Members can mint an NFT indicating that they're a part of this membership.
+* Purchasable using fiat or crypto.
+* Recurring payments, or one-off.
```
Comment (fred-paragraph) [inline src/pages/docs/memberships.md:34]:
```suggestion
* One-off payments will be possible, not just recurring.
```

**Review by fred-paragraph — APPROVED** [review]
Looks good overall, a few nits and typos.

File: src/pages/docs/memberships.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+All publications also have a _membership_ page, accessible at /memberships (for example: https://paragraph.xyz/@blog/memberships). You can share this link with your readers to incentivize them to upgrade and illustrate the perks they'd get.
+
+![Paragraph memberships](/img/membership.png)
+
+## Paid memberships
+
+In order to begin accepting payment for your memberships, you need to connect your Paragraph account with Stripe. You can do so in your [publication settings](https://paragraph.xyz/settings/publication/memberships-token-gating).
+
+After conneting with Stripe you'll be able to create paid memberships and charge a monthly or yearly fee to join.
```
Comment (thefinnomenon) [inline src/pages/docs/memberships.md:26]: typo: connecting

File: src/pages/docs/memberships.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+## Future improvements
+
+This is the first iteration of what's to come. Within the next few months, memberships will be:
+
+* Backed by NFTs. Members can mint an NFT indicating that they're a part of this membership.
+* Purchasable using fiat or crypto.
+* One-off payments will be possible, not just recurring.
+* Memberships can be in limited quantity.
+
+# Importing paid subscribers
```
Comment (thefinnomenon) [inline src/pages/docs/memberships.md:37]: Spacing above this header is too small

File: src/pages/docs/memberships.md [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+title: Memberships
+description: Monetize your Paragraph newsletter with recurring memberships.
+---
+
+Create free or paid memberships to offer benefits to your subscribers and begin monetizing with recurring payments.
+
+---
+
+# Memberships on Paragraph
+
```
Comment (thefinnomenon) [inline src/pages/docs/memberships.md:11]: I think we should mention that you can mark one as the most popular.

**Review by thefinnomenon — APPROVED** [review]

**Pull request merged**